lib/storage/client.js

```diff
@@ -1176,6 +1176,9 @@ class Client extends CryptoConsumer {
       }
     )
     let groupsJson = await validateResponseAsJSON(response)
+    if (groupsJson.groups === null) {
```

Comment:
@carrala @rosasck this is something that should be fixed on the server side as well, unless there is a non 200 level error the api should always return a valid list groups response, even if its empty like below
```json
{
groups: []
}
```
}